src/dbnode/storage/shard.go

```diff
@@ -1910,6 +1910,7 @@ func (s *dbShard) WarmFlush(
 		// where a fileset already exists when we attempt to flush unless there
 		// is a bug in the code.
 		DeleteIfExists: false,
+		FileSetType:    persist.FileSetFlushType,
```

Comment:
Is this just setting it to the default value concretely?

Reply:
Yep this is a noop. Previously it was just using the zero value which corresponds to `persist.FileSetFlushType`. Seemed dangerous to rely on that ordering.